apps/emqx_node_rebalance/src/emqx_node_rebalance_api.erl

```diff
@@ -614,6 +630,14 @@ fields(local_status_enabled) ->
                     required => false
                 }
             )},
+        {connection_eviction_rpc_timeout,
+            mk(
+                emqx_schema:timeout_duration_s(),
```

Comment:
Here, we have `timeout_duration_s`, but in `rebalance_start` it is `timeout_duration_ms`.  Is it intentional?

Reply:
In short, yes, but now I decided to simplify and get ms everywhere.